HomePage.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>UB Tutoring</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type="text/javascript" src="js/modernizr.custom.86080.js"></script>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Alatsi|Anton|Righteous&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Shadows+Into+Light" />
+    <title>UB Tutoring Service</title>
+    <style>
+        .btn {
+            border: none;
+            background-color: inherit;
+            padding: 14px 28px;
+            font-size: 16px;
+            cursor: pointer;
+            display: inline-block;
+        }
+
+        /* Blue */
+        .info {
+            color: dodgerblue;
+        }
+        .info:hover {
+            background: #2196F3;
+            color: white;
+        }
+
+
+    </style>
+</head>
+
+<body>
+
+    <p>This is the Tutor Calendar View Page!!!!!!!</p>
+
+    <button class="btn info">Add/Edit Time Slot</button>
+
+    <p>
+    
+    </p>
+
+    <img src="temp_sched.png" alt="Temporary Schedule">    
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="index.js"></script>
+    <script>
+        $(document).ready(function() {
+
+            displayTime();
+            setInterval(displayTime, 1000);
+
+        });

```

Comment:
if you pull the changes from develop, this part of the code was updated. be sure to git pull before making any changes of your own.